### ISSUE TITLE
Riak TS Download shows AWS S3 Error

### ISSUE THREAD

**Issue by bmalum** [basho/basho_docs#2524]
Hi Guys - just wanted to install Riak and got this message, after I clicked the CentoOS7 Links on this Page: https://docs.riak.com/riak/ts/1.5.2/downloads/index.html  

```
<Error>
<Code>PermanentRedirect</Code>
<Message>
The bucket you are attempting to access must be addressed using the specified endpoint. Please send all future requests to this endpoint.
</Message>
<Endpoint>downloads.riak.com.s3.amazonaws.com</Endpoint>
<Bucket>downloads.riak.com</Bucket>
<RequestId>D8961B128E2ACB14</RequestId>
<HostId>
I/HM7HyqcfZmsWjxB7GvhiMsng3EicPk6UWpGgMFzIXPWc4BOTOPlKJQN6XWoG3EJfgsZ0rC42o=
</HostId>
</Error>
```

**Comment by bmalum** [comment]
FYI: manages to get the download URL for RHEL7 Riak TS 1.5.2  downloads.basho.com.s3.amazonaws.com/basho-internal/riak_ts/1.5/1.5.2/rhel/7/riak-ts-1.5.2-1.el7.centos.x86_64.rpm 

But there are many other broken URLs, sadly => otherwise I would open a MR

**Comment by robotarmy** [comment]
http://downloads.basho.com.s3.amazonaws.com/riak_ts/1.5/1.5.2/osx/10.8/riak-ts-1.5.2-OSX-x86_64.tar.gz

I was able to get OSX above..

Did something terrible happen to BASHO?

**Comment by jaredmorrow** [comment]
> Did something terrible happen to BASHO?

Yes. Some goobers hired some other goobers to run the joint, antics ensued, no more Basho.

The Github Basho org was purchased by another company in the Basho yard sale.

**Comment by Vagabond** [comment]
The tell-all book will drop in 2020 or 2021. Movie rights are available.

**Comment by russelldb** [comment]
https://courts.delaware.gov/Opinions/Download.aspx?id=275470
TL:DR https://www.theregister.co.uk/2018/07/09/basho_damages_20m_misinformation_threats/

**Comment by davaeron** [comment]
http://downloads.basho.com.s3.amazonaws.com/riak_ts/1.5/1.5.2/riak_ts-1.5.2.tar.gz
